package.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<package format="3">
+  <name>osqp-eigen</name>
+  <version>0.6.2</version>
+  <description>Simple Eigen-C++ wrapper for OSQP library </description>
+  <maintainer email="tbd@tbd.tbd">tbd</maintainer>
+
+  <license>LGPLv3</license>
+
+  <url type="website">https://github.com/Robotology/osqp-eigen</url>
+
+  <build_depend>git</build_depend>
+  <!-- The following tags are recommended by REP-136 -->
```

Comment:
Not an expert of catkin/ament, but there is no reference of `ament_cmake` in REP-136: https://www.ros.org/reps/rep-0136.html .

Reply:
Correct, thanks for pointing this out - I wrote this ~14 months ago when my understanding was different. I just fixed it.